### PR TITLE
Improving schema extraction logs

### PR DESCRIPTION
**Pull request by edcdavid**
Improves the logs created by the `ExtractSchemaRequired`, `ExtractSubSchema`, and the  `ExtractMatchingInput` functions.

### CODE REVIEW

File: internal/controllers/utils/utils.go [hunk 1]
```diff
@@ -1243,37 +1243,37 @@ func ExtractSubSchema(mainSchema []byte, subSchemaKey string) (subSchema map[str
 		return subSchema, fmt.Errorf("failed to UnMarshall Main Schema: %w", err)
 	}
 	if _, ok := jsonObject[PropertiesString]; !ok {
-		return subSchema, fmt.Errorf("non compliant Main Schema, missing properties: %w", err)
+		return subSchema, fmt.Errorf(`non compliant Main Schema, missing "properties" section: %w`, err)
 	}
 	properties, ok := jsonObject[PropertiesString].(map[string]any)
 	if !ok {
-		return subSchema, fmt.Errorf("could not cast properties as map[string]any: %w", err)
+		return subSchema, fmt.Errorf(`could not cast "properties" section of schema as map[string]any: %w`, err)
```
Comment (alegacy) [inline internal/controllers/utils/utils.go:1250]: Usually quoting of words in logs is done with a single quote (`'`).

Comment (edcdavid) [reply internal/controllers/utils/utils.go]: Fixed

**Pull request merged**